sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -188,7 +188,6 @@ public partial class SearchOptions : Azure.Search.Documents.SearchRequestOptions
     public partial class SearchRequestOptions
```

Comment:
Empty type. Worth removing?

Reply:
Working on that now. @tg-msft and I were discussing reasons to keep it (none, really) or remove it entirely.